tests/infrastructure/prometheus.go

```diff
@@ -640,3 +638,22 @@ func countReadyAndLeaderPods(pod *k8sv1.Pod, component string) (foundMetrics map
 
 	return foundMetrics, err
 }
+
+func GetKubevirtVMMetrics(virtClient *kubecli.KubevirtClient, pod *k8sv1.Pod, ip string) string {
```

Comment:
You don't need the capital `G` since the function is used only in this file, thus its private to this file scope

Reply:
It's also used on [tests/libmigration/migration.go](https://github.com/kubevirt/kubevirt/pull/11140/files#diff-679184af10d0b8d9703feb21d657e8dcf3b834bacbcab6e3797439ec4f7f19c3).

Reply:
@enp0s3 Am I missing something here?

Reply:
@assafad My concern is that `GetKubevirtVMMetrics` is being used in multiple places, therefore IMO it deserves to live in a libraray. Do you have a library for monitoring? maybe this is the chance to create one

Reply:
1. WDYT about locating `GetKubevirtVMMetrics` in [tests/monitoring/prometheus_utils.go](https://github.com/kubevirt/kubevirt/blob/main/tests/monitoring/prometheus_utils.go)?
2. Can you please show an example of such library that tests use?